coordinator/src/orderbook/db/custom_types.rs

```diff
@@ -130,6 +131,51 @@ impl FromSql<OrderStateType, Pg> for OrderState {
     }
 }
 
+#[derive(Debug, Clone, Copy, PartialEq, FromSqlRow, AsExpression)]
+#[diesel(sql_type = OrderReasonType)]
+pub(crate) enum OrderReason {
```

Comment:
`OrderReason` sounds weird to me. Is it more an `OrderType`?

Reply:
We already have an `OrderType` 🙈

Reply:
How about SubOrderType?